### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by kmr-srbh**
1. Updated the installation instructions referring to the documentation.
2. Reviewed the complete README and fixed sentences and grammar wherever required.

### CODE REVIEW

**Review by certik — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -25,235 +24,143 @@ LFortran, see that project for a list of sponsors.
...
-- Update reference tests
 
-  ```bash
-  python run_tests.py -u --skip-run-with-dbg
-  ```
+        ./build0.sh
+        ./build1.sh
+        ```
 
-## Speed up Integration Tests on MacOS
```
Comment (Thirumalai-Shaktivel) [inline README.md:210]: @kmr-srbh some of the information seems to be removed from the previous README.md. 
Can you please look into it and add them back?

`Speed up Integration Tests on MacOS`
`Examples`
`Troubleshooting on MacOS latest version`
`Tests`

Comment (kmr-srbh) [reply README.md:210]: @Thirumalai-Shaktivel that section was moved to the installation docs.

Comment (Thirumalai-Shaktivel) [reply README.md:210]: Okay, then it's fine.
Yes, I can see it here: https://github.com/lcompilers/lpython/blob/main/doc/src/installation.md

Comment (Thirumalai-Shaktivel) [reply README.md:210]: Updated here: https://github.com/lcompilers/lpython/pull/2537

**Pull request merged**